### PR TITLE
add notebook for debugger profiling blog

### PR DESCRIPTION
**Pull request by mchoi8739**
*Description of changes:*

Adding a Debugger profiling notebook for the blog team.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: sagemaker-debugger/tensorflow_nlp_sentiment_analysis/sentiment-analysis-tf-distributed-training-bringyourownscript.ipynb [hunk 1]
```diff
@@ -0,0 +1,1108 @@
+{
```
Comment (ndodda-amazon) [inline sagemaker-debugger/tensorflow_nlp_sentiment_analysis/sentiment-analysis-tf-distributed-training-bringyourownscript.ipynb:1]: <p>We might need to update this. It looks like the current version of sagemaker SDK in the repo is <a href="https://github.com/aws/sagemaker-python-sdk/blob/master/VERSION" target="_blank">2.18.1</a>. I anticipate this to be upgraded to 2.19.x after the merging of wave 2 features. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1840/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (ndodda-amazon) [inline sagemaker-debugger/tensorflow_nlp_sentiment_analysis/sentiment-analysis-tf-distributed-training-bringyourownscript.ipynb:1]: <p>Do we want to add what kinds of metrics are collected with the below configuration?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1840/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Comment (ndodda-amazon) [inline sagemaker-debugger/tensorflow_nlp_sentiment_analysis/sentiment-analysis-tf-distributed-training-bringyourownscript.ipynb:1]: <p>nit: seems to be two spaces between <code>this</code>  and <code>changes</code>  in <code>Based on this  change the training configuration...</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1840/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='78'/>

Comment (mchoi8739) [reply sagemaker-debugger/tensorflow_nlp_sentiment_analysis/sentiment-analysis-tf-distributed-training-bringyourownscript.ipynb:1]: <p>fixed</p><p> </p>

**Review by ndodda-amazon — CHANGES_REQUESTED** [review]
Looks good, but we should definitely update the minimum version number of sagemaker SDK that has profiling functionality.

File: sagemaker-debugger/tensorflow_nlp_sentiment_analysis/sentiment-analysis-tf-distributed-training-bringyourownscript.ipynb (hunk 1 above)
Comment (mchoi8739) [reply sagemaker-debugger/tensorflow_nlp_sentiment_analysis/sentiment-analysis-tf-distributed-training-bringyourownscript.ipynb:1]: <p>added a sentence</p>

Comment (mchoi8739) [reply sagemaker-debugger/tensorflow_nlp_sentiment_analysis/sentiment-analysis-tf-distributed-training-bringyourownscript.ipynb:1]: <p>fixed</p>

Comment (ndodda-amazon) [reply sagemaker-debugger/tensorflow_nlp_sentiment_analysis/sentiment-analysis-tf-distributed-training-bringyourownscript.ipynb:1]: <p>Just confirmed, sagemaker SDK will be bumped up to 2.19.0 to include the wave 2 features. We should update the minimum version here to be 2.19.0.</p>

Comment (mchoi8739) [reply sagemaker-debugger/tensorflow_nlp_sentiment_analysis/sentiment-analysis-tf-distributed-training-bringyourownscript.ipynb:1]: updated

**Review by ndodda-amazon — APPROVED** [review]
Looks good!

**Review by ndodda-amazon — APPROVED** [review]

**Pull request merged**